extensions/network/network.impl/src/test/java/org/mqnaas/network/impl/reservation/ReservationManagementTest.java

```diff
@@ -53,26 +468,67 @@ public String getId() {
 
 	}
 
-	protected Reservation generateReservation(Date startDate, Date endDate, ReservationState reservationState) {
+	class dummyReservationPerformer implements IReservationPerformer {
+
+		@Override
+		public void performReservation(ReservationResource reservation) throws ResourceReservationException {
+			try {
+				serviceProvider.getCapability(reservation, IReservationAdministration.class).setState(ReservationState.RESERVED);
+			} catch (CapabilityNotFoundException e) {
+				// TODO Auto-generated catch block
+				e.printStackTrace();
+			}
+
+		}
 
-		Period period = new Period(startDate, endDate);
+		@Override
+		public void cancelReservation(ReservationResource reservation) throws ResourceReservationException {
+			try {
+				serviceProvider.getCapability(reservation, IReservationAdministration.class).setState(ReservationState.CANCELLED);
+			} catch (CapabilityNotFoundException e) {
+				// TODO Auto-generated catch block
+				e.printStackTrace();
+			}
+		}
 
-		Set<Device> devices = generateSampleDeviceSet();
+		@Override
+		public void finishReservation(ReservationResource reservation) throws ResourceReservationException {
+			try {
+				serviceProvider.getCapability(reservation, IReservationAdministration.class).setState(ReservationState.FINISHED);
+			} catch (CapabilityNotFoundException e) {
+				// TODO Auto-generated catch block
+				e.printStackTrace();
+			}
+		}
+
+		@Override
+		public void activate() throws ApplicationActivationException {
+			// TODO Auto-generated method stub
+
+		}
+
+		@Override
+		public void deactivate() {
+			// TODO Auto-generated method stub
+
+		}
 
-		return ModelUtils.generateReservation(devices, new HashSet<IRootResource>(), period, reservationState);
 	}
 
-	protected Set<Device> generateSampleDeviceSet() {
+	class dummyTrigger implements Trigger {
```

Comment:
So a not null trigger is required,  but it is not being used in tests... ok, thank you!